visualization/include/pcl/visualization/area_picking_event.h

```diff
@@ -62,13 +67,40 @@ namespace pcl
         {
           if (nb_points_ <= 0)
```

Comment:
Do we even still need `nb_points_`? Here we could simply check whether `cloud_indices_` is empty or whether `indices` is empty after accumulating